lib/yang_translator.c

```diff
@@ -282,15 +287,11 @@ struct yang_translator *yang_translator_load(const char *path)
 error:
```

Comment:
hmm - could we leak 'dnode' in the path up at line 164?

Reply:
hm, seems so too

Reply:
Fixed, thanks :)